source/locale/zh_CN/LC_MESSAGES/tutorial/10_key_features/003_efficient_optimization_algorithms.po

```diff
@@ -0,0 +1,333 @@
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:101
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:201
+msgid "Out:"
+msgstr ""
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:81
+msgid ""
+"If you want to use different samplers for example "
+":class:`~optuna.samplers.RandomSampler` and "
+":class:`~optuna.samplers.CmaEsSampler`,"
+msgstr "如果你希望使用其他采样器，比如 "
+":class:`~optuna.samplers.RandomSampler` 和 "
+":class:`~optuna.samplers.CmaEsSampler`,"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:112
+msgid "Pruning Algorithms"
+msgstr "剪枝算法"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:114
+msgid ""
+"``Pruners`` automatically stop unpromising trials at the early stages of "
+"the training (a.k.a., automated early-stopping)."
+msgstr "``Pruners``  自动在训练的早期（也就是自动化的 early-stopping）终止无望的 trial."
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:116
+msgid "Optuna provides the following pruning algorithms:"
+msgstr "Optuna 提供以下剪枝算法："
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:118
+msgid ""
+"Asynchronous Successive Halving algorithm implemted in "
+":class:`optuna.pruners.SuccessiveHalvingPruner`"
+msgstr ":class:`optuna.pruners.SuccessiveHalvingPruner` 实现的 Asynchronous Successive Halving 算法。"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:120
+msgid "Hyperband algorithm implemented in :class:`optuna.pruners.HyperbandPruner`"
+msgstr ":class:`optuna.pruners.HyperbandPruner` 实现的 Hyperband 算法。"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:122
+msgid ""
+"Median pruning algorithm implemented in "
+":class:`optuna.pruners.MedianPruner`"
+msgstr ":class:`optuna.pruners.MedianPruner` 实现的中位数剪枝算法"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:124
+msgid ""
+"Threshold pruning algorithm implemented in "
+":class:`optuna.pruners.ThresholdPruner`"
+msgstr ":class:`optuna.pruners.ThresholdPruner` 实现的阈值剪枝算法"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:126
+msgid ""
+"We use :class:`optuna.pruners.MedianPruner` in most examples, though "
+"basically it is outperformed by "
+":class:`optuna.pruners.SuccessiveHalvingPruner` and "
+":class:`optuna.pruners.HyperbandPruner` as in `this benchmark result "
+"<https://github.com/optuna/optuna/wiki/%5BUnder-Construction%5D-"
+"Benchmarks-with-Kurobako>`_."
+msgstr "在大多数例子中我们采用的 :class:`optuna.pruners.MedianPruner`, 尽管其性能基本上会被 "
+":class:`optuna.pruners.SuccessiveHalvingPruner` 和 "
+":class:`optuna.pruners.HyperbandPruner` 超过，就像在 `这个基准测试结果 "
+"<https://github.com/optuna/optuna/wiki/%5BUnder-Construction%5D-"
+"Benchmarks-with-Kurobako>`_ 中那样."
```

Comment:
This is not the translation issue, but this link is broken. Could you update it?
```suggestion
"<https://github.com/optuna/optuna/wiki/"
"Benchmarks-with-Kurobako>`_ 中那样."
```